examples/somersaultecu.py

```diff
@@ -1094,7 +1096,8 @@ class SomersaultSID(IntEnum):
             oid=None,
             odx_id=OdxLinkId("somersault.struct.recall.forward_flips_grudgingly_done", doc_frags),
             parameters=somersault_positive_responses["forward_flips_grudgingly_done"].parameters,
-            byte_size=None),
+            byte_size=None,
+            is_visible_raw=None),
```

Comment:
And this is why I hate when they are not used

Reply:
I can add them to whole codebase if you want, but that's for another PR...